packages/forms-web-app/src/dynamic-forms/dynamic-components/boolean-text/index.njk

```diff
@@ -6,19 +6,23 @@
 {% from "govuk/components/textarea/macro.njk" import govukTextarea %}
 
 {% set title = question.pageTitle + " - " + journeyTitle + " - GOV.UK" %}
-{% block pageTitle %}{{ "Error: " + title if errors else title }}{% endblock %}
 
-{% if question.conditional %}
-  {% set conditionalDetails %}
-    {{ govukTextarea({
-      name: question.conditionalId,
-      id: question.conditionalId,
-      label: {
-        text: question.conditional
-      }
-    }) }}
-  {% endset %}
-{% endif %}
+{% set options = [] %}
+
+{% for option in question.options %}
+  {% set options = (options.push({
+    value: option.value,
+    text: option.text,
+    attributes: { "data-cy": "answer-" + option.value },
+    checked: answer == option.value,      
+    conditional: {
+      html: option.conditional.html,
```

Comment:
Interesting that this line doesn't throw an error (might just be Nunjucks behaviour 😄). `option.conditional.html` doesn't exist for option 'no'. Have you checked there are no errors/warnings here when the option has no conditional fields?

Reply:
if a value in nunjucks is undefined/null, nothing is rendered :)